doc/source/_static/scipy.css

```diff
@@ -68,30 +72,56 @@ h3 {
   --pst-color-info: 23, 162, 184;
```

Comment:
I would suggest adding `--pst-color-active-navigation: 1, 50, 67;`, that's using the same "warm black" (#013243) as the h1 headers are using. That could give a bit more uniformity in colors.